packages/js/product-editor/src/components/variations-table/variation-actions-menu/variation-actions-menu.tsx

```diff
@@ -0,0 +1,291 @@
+							renderContent={ () => (
+								<div className="components-dropdown-menu__menu">
+									<MenuGroup
+										label={ __(
+											'List price',
+											'woocommerce'
+										) }
+									>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt( 'regular_price' );
+												onClose();
+											} }
+										>
+											{ __(
+												'Set list price',
+												'woocommerce'
+											) }
+										</MenuItem>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt(
+													'regular_price',
+													__(
+														'Enter a value (fixed or %)',
+														'woocommerce'
+													),
+													( value ) =>
+														addFixedOrPercentage(
+															variation.regular_price,
+															value
+														)?.toFixed( 2 )
+												);
+												onClose();
+											} }
+										>
+											{ __(
+												'Increase list price',
+												'woocommerce'
+											) }
+										</MenuItem>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt(
+													'regular_price',
+													__(
+														'Enter a value (fixed or %)',
+														'woocommerce'
+													),
+													( value ) =>
+														addFixedOrPercentage(
+															variation.regular_price,
+															value,
+															-1
+														)?.toFixed( 2 )
+												);
+												onClose();
+											} }
+										>
+											{ __(
+												'Decrease list price',
+												'woocommerce'
+											) }
+										</MenuItem>
+									</MenuGroup>
+									<MenuGroup
+										label={ __(
+											'Sale price',
+											'woocommerce'
+										) }
+									>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt( 'sale_price' );
+												onClose();
+											} }
+										>
+											{ __(
+												'Set sale price',
+												'woocommerce'
+											) }
+										</MenuItem>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt(
+													'sale_price',
+													__(
+														'Enter a value (fixed or %)',
+														'woocommerce'
+													),
+													( value ) =>
+														addFixedOrPercentage(
+															variation.sale_price,
+															value
+														)?.toFixed( 2 )
+												);
+												onClose();
+											} }
+										>
+											{ __(
+												'Increase sale price',
+												'woocommerce'
+											) }
+										</MenuItem>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt(
+													'sale_price',
+													__(
+														'Enter a value (fixed or %)',
+														'woocommerce'
+													),
+													( value ) =>
+														addFixedOrPercentage(
+															variation.sale_price,
+															value,
+															-1
+														)?.toFixed( 2 )
+												);
```

Comment:
Should we disable this if no sale price is set? Otherwise we get negative values.

Reply:
Well, disabling it is not the solution because the user can enter a greater number, as well for the list price the user can enter a negative value too. So the next step for this is adding validations to prevent those edge cases.
Would be ok if we could have any feedback on disabling the items from the design perspective. I would rather like to hide them if they are non usable. We always can tackle this in a particular issue after having the discussion with Jarek.

Reply:
Yeah we can do this as a follow up.